ote/src/clj/ote/nap/users.sql

```diff
@@ -59,3 +59,7 @@ SELECT group_id
   FROM group_extra
  WHERE key = 'transit-authority?'
    AND value = 'true';
+
+-- name: delete-user!
+-- Delete user from the database
+DELETE FROM "user" as u WHERE u.id = :id;
```

Comment:
What happens if user is in groups or has authored services? Do they need to be deleted first?

Reply:
Yes. This only deletes user. All other data must be deleted separately.
I thought that it would be bad if operator had two users and one of them was removed and all the data from the operator would have been gone.